### ISSUE TITLE
Silent fail when GDB is not installed

### ISSUE THREAD

**Issue by pvanderlinden** [lmacken/pyrasite#62]
When gdb is not installed, pyrasite will not give an error, warning or message, but it won't do anything as well. It would be good to have this as a message like when ptrace is disabled.

**Comment by gw0** [comment]
Yes, this would be great. I had to add `verbose=True` into `inject()` function (as mentioned in #60) to figure that out:

```
# pyrasite-shell 25657

/bin/sh: 1: gdb: not found
```
